### PR TITLE
feat(frontend): enable suggested sepolia tokens

### PR DESCRIPTION
**Pull request by loki344**
# Motivation

With https://github.com/dfinity/oisy-wallet/pull/3337 we enable USDC for users that have not yet touched the "enabling" of this token. This MR implements the same behaviour for its testnet token SepoliaUSDC and ckSepoliaUSDC.

# Changes

- add sepolia tokens of USDC to suggested constants

# Tests

https://github.com/user-attachments/assets/1e77e90e-b87b-408f-a75f-f17406b25c69

local test with `...(nonNullish(LOCAL_CKUSDC_LEDGER_CANISTER_ID) ? [LOCAL_CKUSDC_LEDGER_CANISTER_ID] : []):`

image

### CODE REVIEW

File: src/frontend/src/env/networks.icrc.env.ts [hunk 1]
```diff
@@ -468,3 +468,13 @@ export const ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS = [
 		? [CKERC20_PRODUCTION_DATA.ckUSDC.ledgerCanisterId]
 		: [])
 ];
+
+// Suggested canisters to be enabled by default if the user set no preference
```
Comment (AntonioVentilii) [inline src/frontend/src/env/networks.icrc.env.ts:472]: Maybe here `Additional suggested...` just to comment

File: src/frontend/src/lib/utils/token.utils.ts [hunk 2]
```diff
@@ -64,16 +69,36 @@ export const mapDefaultTokenToToggleable = <T extends Token>({
 }: {
 	defaultToken: T;
 	userToken: TokenToggleable<T> | undefined;
-}): TokenToggleable<T> => ({
-	...defaultToken,
-	enabled:
+}): TokenToggleable<T> => {
```
Comment (AntonioVentilii) [inline src/frontend/src/lib/utils/token.utils.ts:72]: are there any test for this util? shall we create a few, just to be sure, since we are here?

**Review by peterpeterparker — CHANGES_REQUESTED** [review]
Chain Fusion should NOT contain test tokens.

(Blocking just because this is something really important and because Antonio already started reviewing as well)

File: src/frontend/src/lib/utils/token.utils.ts (hunk 2 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/utils/token.utils.ts:72]: Yes. We should create a test first and then provide the change.

**Comment by loki344** [comment]
> Chain Fusion should NOT contain test tokens.
> 
> (Blocking just because this is something really important and because Antonio already started reviewing as well)

Ok, I was not aware of this. Would you mind explaining it for my benefit? 

So I guess I'll close this MR, or do we want to have `SEPOLIA_USDC_TOKEN` suggested and i just remove the ckSepoliaUSDC suggestion?

**Comment by peterpeterparker** [comment]
> Would you mind explaining it for my benefit?

1. It was a product decision.
2. It was also a technical decision:
  a. This facilitated the introduction of the concept of "Chain Fusion" on a codebase that required a network as a mandatory option.
  b. It helps prevent issues on networks where addresses are identical for production and test environments. Specifically, it helps prevent bugs that could, for example, result in transferring tokens on the incorrect network as lont as a comprehensive test suite is not in place.

**Comment by loki344** [comment]
Closing this PR. see https://github.com/dfinity/oisy-wallet/pull/3337 for MR without ckSepolia

**Pull request closed without merging**